src/assets/main.css

```diff
@@ -1,4 +1,7 @@
 @import './base.css';
```

Comment:
Move our custom CSS to below, after all tailwind css.
The order will be:
- Tailwindcss lib(@base, @component, @utilities)
- Your custom CSS